doc/schema_template.md

```diff
@@ -31,6 +31,7 @@ $tag:tool|creator|organization://complexType[@name='Organization']
 $tag:tool|requirements://complexType[@name='Requirements']
 $tag:tool|requirements|requirement://complexType[@name='Requirement']
 $tag:tool|requirements|container://complexType[@name='Container']
+$tag:tool|requirements|resource://complexType[@name='Container']
```

Comment:
What does this do ? How is this different from line 33 ?

Reply:
Ups, stupid copy paste error, should be:

```suggestion
$tag:tool|requirements|resource://complexType[@name='Resource']
```

We already refer to this in the docs for `<requirement>`. 

xref https://github.com/galaxyproject/galaxy/pull/5378